modules/tensor_mechanics/doc/content/modules/tensor_mechanics/FractureIntegrals.md

```diff
@@ -70,9 +70,9 @@ To use the $J$-integral capability in MOOSE, a user specifies a set of nodes alo
 
 The interaction integral method is based on the $J$-integral and makes it possible to evaluate mixed-mode stress intensity factors $K_I$, $K_{II}$ and $K_{III}$, as well as the T-stress, in the vicinity of three-dimensional cracks. The formulation relies on superimposing Williams' solution for stress and displacement around a crack (in this context called 'auxiliary fields') and the computed finite element stress and displacement fields (called 'actual fields'). The total superimposed $J$ can be separated into three parts: the $J$ of the actual fields, the $J$ of the auxiliary fields, and an interaction part containing the terms with both actual and auxiliary field quantities. The last part is called the interaction integral and for a fairly straight crack without body forces, thermal loading or crack face tractions, the interaction integral over a crack front segment can be written [!citep](walters_interaction_2005):
 \begin{equation}
-\bar{I}(s) = \int_V \left[ \sigma_{ij} u_{j,1}^{(aux)} + \sigma_{ij}^{(aux)} u_{j,1} - \sigma_{jk} \epsilon_{jk}^{(aux)} \delta_{1i} \right] q_{,i} \, \mathrm{d}V
+\bar{I}(s) = \int_V \left[ \sigma_{ij} u_{j,1}^{(aux)} + \sigma_{ij}^{(aux)} u_{j,1} - \sigma_{jk} \epsilon_{jk}^{(aux)} \delta_{1i} \right] q_{,i} \, \mathrm{d}V + \int_V \left[ \epsilon^*_{ij,1} \sigma_{ij}^{(aux)} - b_i u_{i,1}^{(aux)} \right] q \, \mathrm{d}V
 \end{equation}
-where $\sigma$ is the stress, $u$ is the displacement, and $q$ is identical to the $q$-functions used for $J$-integrals. This is the formulation of the interaction integral used currently in MOOSE. It is not recommended to use the interaction integral method in analyses where body forces or crack face tractions are important, as the code does not include the necessary correction terms in the integral.
+where $\sigma$ is the stress, $u$ is the displacement, and $q$ is identical to the $q$-functions used for $J$-integrals. The last two terms in this equation account for the effects of the gradient of the eigenstrain, $\epsilon^*$, and for the effects of the body force, $b$. If the eigenstrain is a thermal strain, its gradient is computed using the gradient of the temperature field and the temperature derivative of the eigenstrain. In that case, the user supplies the temperature and eigenstrain name through the `temperature` and `eigenstrain_name` parameters, respectively. If the eigenstrain is from another source, its gradient must be computed by the eigenstrain material model in the form of a RankThreeTensor object, whose name is specified for the fracture integrals using the `eigenstrain_gradient` parameter.
```

Comment:
```suggestion
where $\sigma$ is the stress, $u$ is the displacement, and $q$ is identical to the $q$-functions used for $J$-integrals. The last two terms in this equation account for the effects of the gradient of the eigenstrain, $\epsilon^*$, and for the effects of the body force, $b$. If the eigenstrain is a thermal strain, its gradient is computed using the gradient of the temperature field and the temperature derivative of the eigenstrain. In that case, the user supplies the temperature and eigenstrain name through the [!param](/DomainIntegral/temperature) and [!param](/DomainIntegral/eigenstrain_name) parameters, respectively. If the eigenstrain is from another source, its gradient must be computed by the eigenstrain material model in the form of a RankThreeTensor object, whose name is specified for the fracture integrals using the [!param](/DomainIntegral/eigenstrain_gradient) parameter.
```

We should consistently use `[!param](syntax)`. In the contect of this md it was not immediately clear what object/action these parameters refer to.

As a side note `/DomainIntegral` should really be `/Modules/TensorMechanics/DomainIntegral`!

Reply:
Thanks, I didn't realize we had a standardized way of doing that -- I'll make that change. I agree about moving this under Modules/TensorMechanics. This was around before that was a thing. Let's save that for later, though!